scania-digital-design-system/tegel
# Fix/dts 1958 header align slots

**Describe pull-request**  

BREAKING

###Aligns prop naming:
 - button-label -> label
 - button-icon -> icon
 - image -> thumbnail
 - heading -> header
 - subheading -> subheader

###Aligned props:
 - buttonLabel -> label
 - heading -> header
 - subheading -> subheader

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1958

**How to test**  
1. Go to storybook
2. Check in Components -> Header / Patterns -> Navigation
3. Check the styling and usage of the slot and props affected.
